src/system_tests/Test_wsElimination.h

```diff
@@ -0,0 +1,189 @@
+/*********************                                                        */
+/*! \file Test_wsElimination.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Guy Amir
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** [[ Add lengthier description here ]]
+
+**/
+
+
+#ifndef MARABOU_TEST_WSELIMINATION_H
+#define MARABOU_TEST_WSELIMINATION_H
+
+
+
+#include <cxxtest/TestSuite.h>
+#include "AcasParser.h"
+#include "Engine.h"
+#include "FloatUtils.h"
+#include "GlobalConfiguration.h"
+#include "InputQuery.h"
+#include "Preprocessor.h"
+#include "QueryLoader.h"
+#include "MarabouError.h"
```

Comment:
We sort the includes alphabetically. MarabouError is out of place

Reply:
OK, I originally copied from the acas system test so I fixed it in both mine and the acas test now